### DISCUSSION TITLE
AutoOrtho Connection Errors

### DISCUSSION THREAD

**Discussion by JPVB** [kubilus1/autoortho#133] [Q&A]
Installed AutoOrtho 0.4.2 from GitHub and then download the Europ Ortho Set in my XP12/Custom Scenery. Then Run AutoOrtho and starting XP12 at Stockholm with the Europ AutoOrtho.

I get download errors, see attached pictures.

Any help appreciated ! Thanks.
[](url)

  
![unknown_2023 05 08-12 41](https://github.com/kubilus1/autoortho/assets/7436312/cef6a13c-2ad7-423b-89b3-5c92b4d5ed3a)
![unknown_2023 05 08-12 54](https://github.com/kubilus1/autoortho/assets/7436312/aaa5e948-d836-4112-925f-4458a06b8672)
![unknown_2023 05 08-13 22](https://github.com/kubilus1/autoortho/assets/7436312/1a679dcd-c618-4488-8007-14af92ee2fd9)

**Comment by RyanChon1113** [comment]
I have this problem too, please help

**Reply by kubilus1** [reply]
Looks like throttling from bing.

**Comment by Medeir0s95** [comment]
I have a same problem. Could that my titles can't a loading in my simulator?? 

![image](https://github.com/kubilus1/autoortho/assets/137718884/f06ec946-6412-40fb-81b6-639e38ce9b76)

**Reply by kubilus1** [reply]
A blue region can mean just a slow loading DSF file.  I don't see as many errors in your log.

**Reply by Medeir0s95** [reply]
What do you mean 'bout DSF? How can i solve that?

**Reply by Medeir0s95** [reply]
I ajust the MaxWait to 3.0 and this not a solve my problem. I don't know what's happened. With you wanna do know 'bout my PC, that's my config:

I59400F
B365 PROVDH MSI
16Gb RAM 2666
RTX3060 
SSD NVME M2 1TB Kingston 2100Mb for reading and 1700mb for recording

**Comment by kubilus1** [comment]
In your case it may be slow response from the servers.  For instance using
bing servers from outside the US may be slower.

**Reply by Medeir0s95** [reply]
What do you recomended me??

**Reply by dpdurst** [reply]
If you read the screenshot you posted it specifically states there are timeout issues accessing the server. That is on your end or in-between your end and the server. I as well see it once in a while but not much. You can try a VPN to see if your ISP is throttling you or something else is acting up. But outside of this not sure what more is going to fix it.
